web/themes/custom/cybersecurity/templates/block/block--inpagenavigationblock.html.twig

```diff
@@ -0,0 +1,37 @@
+{#
```

Comment:
This is pretty much the core (stable) block template... I'd create a default block template using this markup and remove this template.